lambda/aws-qa-appsync-opensearch/question_answering/src/llms/text_generation_llm_selector.py

```diff
@@ -50,9 +50,9 @@ def get_llm(callbacks=None):
 
     return Bedrock(**kwargs)
 
-def get_embeddings_llm():
+def get_embeddings_llm(model_id):
```

Comment:
- since the model id is a user input, is there a check to verify if the model exists ? 
- also, are all the models in Bedrock available through that Langchain API (maybe need a version update) ?